### PR TITLE
Unique

### PR DESCRIPTION
**Pull request by benjub**
A few minor commits concerning comments. This begins with the commit `section 'empty set': clarify...`

### CODE REVIEW

**Comment by benjub** [comment]
New contributor: the three `conventions-*` statements end thus:
```
       </HTML>

       (Contributed by the Metamath team, 27-Dec-2016.)  Date of last revision.
       (Revised by the Metamath team, 22-Sep-2022.)
       (Proof modification is discouraged.)  (New usage is discouraged.) $)
```
Hope this is faithful to https://groups.google.com/g/metamath/c/yZPXFNGSHZ8 (ping @digama0 and @david-a-wheeler) I hope David is ok with the new "contributed by", which I think gives more "power" to the conventions, but of course I can restor the previous "contributed by".

**Comment by benjub** [comment]
Ok, I'm done. Enough nitpicking.

**Review by jkingdon — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -256213,23 +256222,26 @@ univariate polynomial evaluation map function for a (sub)ring is a ring
...
-of this expression as definition ~ df-zring of the ring of integers.
-
-Remark: Instead of using the symbol "ZZrng" analogous to ` CCfld ` used for the
-field of complex numbers, we have chosen the version with an "i" to indicate
-that the ring of integers is a unital ring, see also Wikipedia ("Rng
-(algebra)", 9-Jun-2019, ~ https://en.wikipedia.org/wiki/Rng_(algebra) ).
+
+  According to Wikipedia ("Integer", 25-May-2019,
+  ~ https://en.wikipedia.org/wiki/Integer ) "The integers form a unital ring
+  which is the most basic one, in the following sense: for any unital ring,
```
Comment (wlammen) [inline set.mm:256228]: line wrapping:  Why was the "there" moved to the next line?

Comment (benjub) [reply set.mm:256228]: I fixed the indentation of this section comment (two spaces), hence the line wrapping.

**Pull request merged**